### PR TITLE
Release 0.8.0+1.1.1

### LINKED ISSUE #41: Upgrade to MoltenVK v1.1.1
That was just released with nice Vulkan 1.1 support and Apple Silicon fixes

https://github.com/KhronosGroup/MoltenVK/releases/tag/v1.1.1

### PR DESCRIPTION
**Pull request by MarnixKuijs**
build.rs now deals with XCFrameworks properly. MoltenVK version upgrade to 1.1.1. Support for aarch64 version MoltenVK.

Fixes #41 and #36

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -15,6 +15,9 @@ documentation = "https://docs.rs/ash-molten"
 [dependencies]
 ash = "0.31"
 
+[build-dependencies]
+xcframework = { git = "https://github.com/EmbarkStudios/xcframework", branch = "main" }
```
Comment (MaikKlein) [inline Cargo.toml:19]: Probably a good idea to pin this to a specific commit, or release a version of xcframework on crates.io.

Comment (repi) [reply Cargo.toml:19]: Not just a good idea, you have to publish that crate on crates.io in order to publish this crate on crates.io

**Review by repi — CHANGES_REQUESTED** [review]
The `xcframework` crate needs to be published for this to work and be merged.

Have you tested these changes on `aarch64-apple-darwin` also? As it changes the fairly complex loading I did before

**Comment by MarnixKuijs** [comment]
Will be working on improving xcframework, before it can get published. So this commit will be on hold for now.

**Comment by VZout** [comment]
Any chance we can update to MoltenVK latest instead of 1.1.1 for various rust-gpu related fixes?

**Comment by VZout** [comment]
@MarnixKuijs Would you like some help on this PR? It's blocking me on a couple of issues.

**Comment by MarnixKuijs** [comment]
@VZout yeah sure, the only thing that needs to happen is open sourcing https://github.com/EmbarkStudios/xcframework, but if that takes too much time you can also just take the code from that crate and add it to ash-molten.

**Comment by repi** [comment]
Converting to draft until it works and is ready to be re-reviewed

**Comment by VZout** [comment]
Spend a whole lot of time trying to figure out why the builds were failing turns out I exceeded the rate limit by my many commits. 🤦

File: Cargo.toml (hunk 1 above)
Comment (VZout) [reply Cargo.toml]: Merged xcframework into ash-molten as a module.

**Comment by VZout** [comment]
Merged `xcframework` into this repo as a module. Build fails because of Github API rate limiting.

**Comment by repi** [comment]
Replaced by #45

**Pull request closed without merging**